src/App.tsx

```diff
@@ -5,7 +5,15 @@ import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+
+  const addMoviesList = (movie: Movie) => {
```

Comment:
```suggestion
  const addMovie = (movie: Movie) => {
```
Or if you want to highlight `list` word => `addMovieToList`